_i18n/en.yml

```diff
@@ -501,6 +501,15 @@ faq:
   avuln: >
     The Monero community has created a series of videos called "Breaking Monero", where potential Monero vulnerabilities are explored and discussed. There are 14 videos, with each exploring a different subject. Check out <a href="https://www.monerooutreach.org/breaking-monero/">the videos on monerooutreach.org</a>.
   vulnspotify: Available on Spotify as podcast
+  qcriminals: Is Monero for criminals?
+  acriminals: No. Monero is permissionless, peer-to-peer and decentralized. This means Monero developers and community have no control over how is used and who uses the technology. Monero is a tool in the hands of the public, very much like a kitchen knife or the US dollar, both of which are currently used to commit or fund horrible crimes, like terrorist attacks.
```

Comment:
```suggestion
  acriminals: No, Monero was created for everyone who values privacy, and not everyone who values privacy is a criminal. Monero is a permissionless, peer-to-peer and decentralized currency. This means anyone can participate on the network, and there is not a central authority to control how it is used and who uses the technology. Monero is a neutral tool in the hands of the public, very much like a kitchen knife, which can be used to slice a loaf of bread or stab an innocent person, or the US dollar, which can be used to donate do charity projects or fund terrorist attacks.
```

Reply:
This is a bit long, would be better to trim it, but if people want the change i'm ok with it.